include/sway/desktop/launcher.h

```diff
@@ -3,10 +3,30 @@
 
 #include <stdlib.h>
 
-struct sway_workspace *workspace_for_pid(pid_t pid);
+struct launcher_ctx {
+	pid_t pid;
+	char *name;
+	struct wlr_xdg_activation_token_v1 *token;
+	struct wl_listener token_destroy;
 
-void launcher_ctx_create(pid_t pid);
+	bool activated;
```

Comment:
This is unused.

Reply:
For my reference, this is first assigned to in e8deba2b (launcher: export xdga tokens [...])

TBH I can't 100% remember why I introduced this other than I think I was having trouble with some clients (firefox) that would map two surfaces at the same time with the same token. My intention is to make the token inert after I match something with it. I added this bool after my first approach didn't work in a later commit, and apparently the rebase isn't as simple as I had hoped if I want the history to be correct.